src/standalone/Vegetation/optimality_farquhar.jl

```diff
@@ -112,14 +112,25 @@ function OptimalityFarquharParameters{FT}(;
 end
 
 """
-    OptimalityFarquharModel{FT} <: AbstractPhotosynthesisModel{FT}
+    OptimalityFarquharModel{FT,
```

Comment:
Does this one need something like the following? I saw you added it to some of the other canopy parameterizations.

`Base.eltype(::BigLeafEnergyParameters{FT}) where {FT} = FT`

Reply:
If I didn't add it, it means that it is not used anywhere at the moment. We can add it if needed.